### PR TITLE
added mathjax for latex support

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by jacobtomlinson** [comment]
I like this. However it does introduce a new dependancy and request which is loaded on every page.

I think it would be sensible to have this an an optional include, that you can enable un the frontmatter of an article. This could either look like:

```
mathjax: true
```

or 

```
javascript:
 - mathjax
```

or similar.

The second allows for future things like D3 to also be added but also adds complication. I'll leave it to you to come up with something.

**Comment by ghost** [comment]
Hello, Jacob!

I added, that mathjax only gets loaded, when it's needed. Thanks to kramdown: http://kramdown.gettalong.org/math_engine/mathjax.html

File: _includes/head.html [hunk 1]
```diff
@@ -36,4 +36,10 @@
 <!-- Go to www.addthis.com/dashboard to customize your tools -->
 <script type="text/javascript" src="//s7.addthis.com/js/300/addthis_widget.js#pubid={{ site.addthis_id }}"></script>
 {% endif %}
+
+{% if page.content contains "<script type="math/tex" %}
```
Comment (jacobtomlinson) [inline _includes/head.html]: You should probably use single quotes, you have a quote collision here.

**Comment by jacobtomlinson** [comment]
Looks good. Although it has a bug, see commit comment.

**Comment by ghost** [comment]
fixed

**Comment by jacobtomlinson** [comment]
Great! That's a really neat fix.

Could you finally just add a example of this in the [Carte Noire in Action](http://carte-noire.jacobtomlinson.co.uk/jekyll/2014/06/10/carte-noire-in-action/) article so that people can see it in use.

**Comment by ghost** [comment]
Done!

**Pull request merged**